### PR TITLE
ENH : Improve test coverage in logger.js

### PR DESCRIPTION
**Pull request by marco-souza**
This branch has some QA enhancements

### CODE REVIEW

File: src/utils/__tests__/logger.test.js [hunk 1]
```diff
@@ -44,4 +44,34 @@ describe("LoggerFactory", () => {
 
         expect(global.console.error).toBeCalled();
     });
+
+    it("should resturn a empty string on _getLevelColor a unexistent level ", () => {
```
Comment (brunomacf) [inline src/utils/__tests__/logger.test.js]: Take a look on this small spelling errors and maybe you should put a "with" after _getLevelColor to be more understandable (like "should return an empty string on _getLevelColor with unexistent level as param" for example). I know i'm a pain in the ass :)

File: src/utils/__tests__/logger.test.js [hunk 2]
```diff
@@ -44,4 +44,34 @@ describe("LoggerFactory", () => {
...
+        const Logger = new LoggerFactory({
```
Comment (brunomacf) [inline src/utils/__tests__/logger.test.js]: As far as i remember LoggerFactory constructor accepts just one string parameter, say the top level scope of logging. So this line should be just `const Logger = new LoggerFactory("global")` for example.

File: src/utils/logger.js [hunk 3]
```diff
@@ -41,10 +41,7 @@ class LoggerFactory {
      * @returns {Logger} A new logger instance.
      */
     create(scopeName: string): Logger {
-        return new Logger({
-            moduleName: this.moduleName,
-            scopeName
-        });
+        return new Logger( this.moduleName, scopeName );
```
Comment (brunomacf) [inline src/utils/logger.js]: Passing an object as parameter is preferable when a function should receive more then one parameter. This is the javascript way to put "labels" on parameters and prevent things like `callThatFunctionWith('lele', 'lolo')` where it's impossible to understand what `lele` and `lolo` stands for. This is just a convention of course, maybe you could agree in have object params when function should receive 3 or more params. What do you think?

File: src/utils/__tests__/logger.test.js (hunk 1 above)
Comment (marco-souza) [reply src/utils/__tests__/logger.test.js]: Done

File: src/utils/__tests__/logger.test.js (hunk 2 above)
Comment (marco-souza) [reply src/utils/__tests__/logger.test.js]: Done

File: src/utils/logger.js (hunk 3 above)
Comment (marco-souza) [reply src/utils/logger.js]: Yes, I think you are right. The changes are done.

File: src/utils/logger.js [hunk 4]
```diff
@@ -172,3 +172,6 @@ class Logger {
 
 // Exports
 export default LoggerFactory;
+export {
```
Comment (brunomacf) [inline src/utils/logger.js]: How about just `export Logger` and then you can use the syntax `import LoggerFactory, {Logger} from './logger'` in your tests. Check if this `export Logger` syntax works please and give me a feedback.

Comment (marco-souza) [reply src/utils/logger.js]: So are you saying to remove the { } and change my import in the test file, right?

Comment (marco-souza) [reply src/utils/logger.js]: Also, the `export Logger` syntax just work before the object definition, in the end of the file don't. An alternative is:
```
export {
    LoggerFactory as default,
    Logger,
};
```

What do you think?

Comment (brunomacf) [reply src/utils/logger.js]: Totally great!! Awesome! :)

**Pull request merged**